tests/e2e/testdata/operator.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
```

Comment:
why not use sed to generate this file on the fly as to avoid duplication of the manifests which may become inconsistent over time?